i18n/locales/en.ts

```diff
@@ -1711,7 +1717,7 @@ const translations = {
         item_two: 'Fee',
         item_two_b: 'Satoshis',
         item_three: 'Weight',
-        item_three_b: 'Weight units (WU)',
+        item_three_b: 'Weighted units (WU)',
```

Comment:
it is actually "weight units" ... did you find a source that calls them "weighted" ?